src/test/resources/posters/tests/smoke/RegisteredOrderTest.xml

```diff
@@ -4,12 +4,17 @@
         <data key="email">john@doe.com</data>
         <data key="password">topsecret</data>
         <data key="firstname">John</data>
-        <data key="lastname">Doe</data>
+        <data key="name">John Doe</data>
```

Comment:
can we adjust the POJO to have a clear difference between `firstname` and `lastname`.
please revert back to last name at the relevant places.